src/main/resources/README.md

```diff
@@ -1,6 +1,10 @@
 # Resources
 This folder contains resources that are included in the Gobra jar file.
 
+## Built-Ins
+The `.gobra` files in the `builtin` folder form a package and thus need to have the same package clause and be located in the top-level directory (i.e. subdirectories are ignored).
+Members declared in these files will be available without any import in the file being verified and can be accessed without qualifier.
+
```

Comment:
To add to what @ArquintL said, it is worth noting that this is the same approach as in the Go compiler/std library. All definitions in the `builtin` package  are available without an explicit import and do not require qualification.

Reply:
However, I couldn't find a source that the Go compiler actually handles these built-ins like we do it here. The documentation of the `builtin` package [here](https://golang.org/pkg/builtin/) only mentions that `builtin` is a package for documentation purposes